tests/unit/test_file_operations.py

```diff
@@ -33,6 +33,7 @@ def setUp(self):
         self.test_file = str(self.workspace.get_path("test_file.txt"))
         self.test_file2 = "test_file2.txt"
         self.test_directory = str(self.workspace.get_path("test_directory"))
+        self.test_nested_file = os.path.join(self.test_directory, self.test_file)
```

Comment:
I was trying to make a nested file so it would test creating the directory, I changed it to:

```python
self.test_nested_file = str(self.workspace.get_path("nested/test_file.txt"))
```